MIT.md

```diff
@@ -0,0 +1,7 @@
+## Copyright 2023, Victor Miñope
+
+Permission is hereby granted, free of charge, to any person obtaining a copy of this [APP TYPE] and associated documentation files, to deal in the [APP TYPE] without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the [APP TYPE], and to permit persons to whom the [APP TYPE] is furnished to do so, subject to the following conditions:
```

Comment:
[Optional] Kindly include the name of the project for which is licence is being applied to